airflow/models/operator.py

```diff
@@ -16,11 +16,24 @@
 # specific language governing permissions and limitations
 # under the License.
 
+import datetime
 from typing import Union
 
+from airflow.configuration import conf
 from airflow.models.baseoperator import BaseOperator
 from airflow.models.mappedoperator import MappedOperator
+from airflow.utils.trigger_rule import TriggerRule
+from airflow.utils.weight_rule import WeightRule
 
 Operator = Union[BaseOperator, MappedOperator]
 
 __all__ = ["Operator"]
+
+DEFAULT_OWNER = conf.get("operators", "default_owner")
+DEFAULT_POOL_SLOTS = 1
+DEFAULT_PRIORITY_WEIGHT = 1
+DEFAULT_QUEUE = conf.get("operators", "default_queue")
+DEFAULT_RETRIES = conf.getint("core", "default_task_retries", fallback=0)
+DEFAULT_RETRY_DELAY = datetime.timedelta(seconds=300)
+DEFAULT_WEIGHT_RULE = conf.get("core", "default_task_weight_rule", fallback=WeightRule.DOWNSTREAM)
+DEFAULT_TRIGGER_RULE = TriggerRule.ALL_SUCCESS
```

Comment:
These had to be moved because I think mypy got in to an import loop and couldn't work out the type of these variables!